### PR TITLE
test derive Arbitrary

### PR DESCRIPTION
**Pull request by bobzhang**

### CODE REVIEW

File: quickcheck/arbitrary_test.mbt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+struct H {
+  x : Int
+  y : Int
+} derive(@quickcheck.Arbitrary, Show)
+
```
Comment (bobzhang) [inline quickcheck/arbitrary_test.mbt:19]: Note the user experience is worth further investigation

**Review by CAIMEOX — APPROVED** [review]

File: builtin/uinit64.mbt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Copyright 2024 International Digital Economy Academy
```
Comment (gmlewis) [inline builtin/uinit64.mbt:1]: Did you want this filename to be `builtin/uint64.mbt` ?

**Comment by bobzhang** [comment]
@lijunchen can you investigate why the CI hang forever

**Comment by lijunchen** [comment]
> @lijunchen can you investigate why the CI hang forever

This is because `random/splitmix/random.mbt::pop_count` causes infinity loop in js backend

I submit a temporary fix in https://github.com/moonbitlang/core/pull/984

**Pull request merged**